graphistry/plotter.py

```diff
@@ -653,3 +653,49 @@ def _make_vgraph_dataset(self, edges, nodes, name):
         dataset = vgraph.create(elist, filtered_nlist, sources, dests, nodeid, node_map, name)
         dataset['encodings'] = encodings
         return dataset
+
+
+    def bolt(self, driver):
```

Comment:
If you put python comments here, then a notebook user can do `help(g.bolt)` to learn about how to use this

Reply:
added some comments. feel free to update them with something more user friendly. I'm not sure how to appease data scientists with these comments.